Singularity/Singularity/Screen/BarIWindowItem.cs

```diff
@@ -4,36 +4,68 @@
 
 namespace Singularity.Screen
 {
+    /// <summary>
+    /// A Bar. Can be used to divide sections in a WindowObject
+    /// </summary>
     internal sealed class BarIWindowItem : IWindowItem
     {
         // color of bar
         private readonly Color mColor;
 
-        public BarIWindowItem(Vector2 size, Color color)
+        /// <summary>
+        /// Creates a simple IWindowItem that is just a bar
+        /// </summary>
+        /// <param name="width">the bar's width</param>
+        /// <param name="color">the bar's color</param>
+        public BarIWindowItem(float width, Color color)
         {
-            Position = new Vector2(0, 0);
-            Size = size;
+            Position = Vector2.Zero;
+            Size = new Vector2(width, 1);
             mColor = color;
             ActiveInWindow = true;
         }
 
+        /// <inheritdoc />
+        /// <summary>
+        /// </summary>
```

Comment:
empty comments ...

Reply:
I thought the inheritdoc needed the summary. Apparently not